addons/l10n_in/models/company.py

```diff
@@ -4,6 +4,12 @@
 class ResCompany(models.Model):
     _inherit = 'res.company'
 
+    l10n_in_edi_production_env = fields.Boolean(
+        string="Indian Electronic Processing Is production",
+        help="Enable the use of production credentials",
+        groups="base.group_system",
+        default=True
```

Comment:
```suggestion
```
We did not have the default production to True before and it is better to let people explicitly say it is going to official to avoid any accidents. Or what is the reason for this change?

Reply:
The reason is too many tickets related to this "users don't tick this and try to validate credentials."
And so the call goes to the testing server and it's failed to validate.

Also, this new service does not work without production.
And when we create a database with demo data then we False this for that company.

Reply:
Is not there something we can do usability wise to make them not forget to check it?

Reply:
It's not a big deal we can improve it later